### DISCUSSION TITLE
Setting Searx as default search engine in Android Chromium based

### DISCUSSION THREAD

**Discussion by theQureshi** [searx/searx#2569] [General]
Hi awesome fellas!

I just installed Searx on a VPS and everything is working perfectly fine except OpenSearch on Android Chromium based browsers.

I have tried going through GitHub issues but I couldn't find anything. Can anybody help?

- Installation followed: Manual (not Docker)
- Installation path: sub.domain.com/searx
- base_url in settings.yml: "https://sub.domain.com/searx"
- Search method: Default is POST but I have tried GET
- Browser: Brave (Chromium based)
- More info: 
-- This seems like a known issue: #1666, #2317, and others.
-- Seems like it was fixed in #2132

Thanks :)

**Comment by unixfox** [comment]
Did you install Searx from the master branch?

**Comment by return42** [comment]
> base_url in settings.yml: "https://sub.domain.com/searx"

Chromium does not like path names in the URL. On my intro page at https://darmarit.org/ I added the following line:

```html
<link title="searx" type="application/opensearchdescription+xml" rel="search" href="/searx/opensearch.xml">
```

When I want to add my engine to a chromium browser I navigate to the root URL and then I open the preferences in Chromium, where I now can find a entry of my search-engine .. 

This was the only robust solution I found.

**Reply by theQureshi** [reply]
Thank you for your input.

Where did you add the above-mentioned line of code? Could you please direct me to the file name?

**Reply by return42** [reply]
In the intro page .. when your root URL is https://sub.domain.com .. the (HTML) page that is shown under this URL has to include the line. 

On my host a apache is listening at the public URL https://darmarit.org/ and in my apache configuration the DocumentRoot is:

```apache
DocumentRoot /srv/http
<Directory /srv/http>
    AllowOverride None
    Require all granted
</Directory>
```

With, the intro page is located at `/srv/http/index.html`. This is typical for archlinux, but you might have a complete different setup .. do you use nginx? ..

Ask yourself, which page is behind your "https://sub.domain.com"? This is the intro page, the _DocumentRoot_.

**Reply by theQureshi** [reply]
Thanks, I will look into this.
